appserver/resources/resources-connector/src/main/java/org/glassfish/resources/admin/cli/ResourceConstants.java

```diff
@@ -259,6 +259,7 @@ public final class ResourceConstants {
     public static final String THREAD_LIFETIME_SECONDS = "thread-lifetime-seconds";
     public static final String TASK_QUEUE_CAPACITY = "task-queue-capacity";
     public static final String MAX_ASYNC = "max-async";
+    public static final String SKIP_CLIENT_INFO_VALIDATION = "skip-client-info-validation";
```

Comment:
You should move this up to under ` // JDBC Connection pool` - where it is you've put it under `// concurrent resource objects`

Reply:
Resolved